assets/javascripts/initializers/code-bytes.js

```diff
@@ -145,7 +145,12 @@ function initializeCodeByte(api) {
       const params = [];
       params.push(`lang=${language}`);
       params.push(`text=${Base64.encodeURI(text)}`);
-      params.push(`mode=${isPreview ? 'compose' : 'view'}`);
+
+      params.push(`client-name=forum`);
+      params.push(`page=${document.location.href}`);
```

Comment:
We should base64 encode this and/or take location.host+location.pathname instead of the full href. Otherwise, URL parameters for the parent window end up as URL parameters to the iframe, which is not what we want.